src/sentry/search/events/fields.py

```diff
@@ -2178,6 +2178,42 @@ def __init__(self, dataset: Dataset, params: ParamsType):
                     ),
                     default_result_type="integer",
                 ),
+                SnQLFunction(
+                    "count",
+                    snql_aggregate=lambda _, alias: Function(
+                        "count",
+                        [],
+                        alias,
+                    ),
+                    default_result_type="integer",
+                ),
+                SnQLFunction(
+                    "last_seen",
+                    snql_aggregate=lambda _, alias: Function(
+                        "max",
+                        [self.column("timestamp")],
+                        alias,
+                    ),
+                    default_result_type="integer",
+                ),
+                SnQLFunction(
+                    "latest_event",
+                    snql_aggregate=lambda _, alias: Function(
+                        "argMax",
+                        [self.column("id"), self.column("timestamp")],
+                        alias,
+                    ),
+                    default_result_type="integer",
```

Comment:
```suggestion
                    default_result_type="string",
```

Reply:
This SnQLFunction class subclasses the DiscoverFunction class. So `snql_aggregate` replaces one of these: `aggregate` or `transform` or `conditional_transform` or `column` on the functions that appear in the original [FUNCTIONS](https://github.com/getsentry/sentry/blob/master/src/sentry/search/events/fields.py#L1525) dictionary. We'll still want to copy over all the other args like `default_result_type`, `redundant_grouping` and their values that appear in the [original version](https://github.com/getsentry/sentry/blob/master/src/sentry/search/events/fields.py#L1591-L1592)